dashboard/app/views/levels/editors/fields/_applab_data.html.haml

```diff
@@ -1,6 +1,6 @@
 %h1.control-legend{data: {toggle: "collapse", target: "#data"}}
   Data
-#data.in.collapse
+#data.collapse
```

Comment:
do you happen to know why removing `in` works here?

Reply:
My understand is that 'in' means you are in the collapser and so then its open. I just looked up the old PR I reverted a while back because people wanted this removed and so I haven't looked at the documentation in a while. Do you know of another way?